src/Forms/bilinearform.jl

```diff
@@ -0,0 +1,42 @@
+default_index_map = prime
+default_bra_vertex_map(v) = (v, "bra")
+default_ket_vertex_map(v) = (v, "ket")
+default_operator_vertex_map(v) = (v, "operator")
+default_tno_constructor(s::IndsNetwork) = delta_network(s)
+
+struct BilinearForm{V,KetMap,BraMap,OperatorMap,SpaceMap} <:
+       AbstractForm{V,KetMap,BraMap,OperatorMap,SpaceMap}
+  tn::ITensorNetwork{V}
+  bra_vs_map::BraMap
+  ket_vs_map::KetMap
+  operator_vs_map::OperatorMap
+  dual_space_map::SpaceMap
```

Comment:
```suggestion
  bra_vertex_map::BraMap
  ket_vertex_map::KetMap
  operator_vertex_map::OperatorMap
  dual_index_map::IndexMap
```
In ITensors.jl we use `space` to refer to the part of an Index without the identifiers (the ID, prime levels, and tags).